dnf/conf/substitutions.py

```diff
@@ -37,24 +37,28 @@ def _update_from_env(self):
         numericvars = ['DNF%d' % num for num in range(0, 10)]
         for key, val in os.environ.items():
             if ENVIRONMENT_VARS_RE.match(key) or key in numericvars:
+                self[key.lstrip("DNF_VAR_")] = val
+            elif key in numericvars:
                 self[key] = val
 
-    def update_from_etc(self, installroot):
+    def update_from_etc(self, installroot, varsdir=("/etc/yum/vars/", "/etc/dnf/vars/")):
```

Comment:
The default value of the varsdir in libdnf/conf/ConfigMain.cpp is in different order: "/etc/dnf/vars", "/etc/yum/vars". It should be unified.

Reply:
Sorry, I missed PR   https://github.com/rpm-software-management/libdnf/pull/803 :)